assets/js/components/DataBlock/index.js

```diff
@@ -106,8 +107,17 @@ const DataBlock = ( {
 						googlesitekit-data-block__title
 					"
 				>
-					{ title }
-					{ badge }
+					{ badge === true ? (
+						<Badge
+							className="googlesitekit-badge--hidden"
+							label="X"
```

Comment:
Will the hidden style prevent this from being detected by a screen reader? We might want to add an aria-hidden on this.

Also, how about we use a more descriptive label even though it's not being shown?
```suggestion
							label="(placeholder)"
```

Reply:
Thanks @aaemnnosttv, I believe the hidden style will hide it from screen readers, but I think it's worthwhile adding `aria-hidden` to be safe, which I have done.

Regarding the label, I considered a more verbose placeholder but opted for the single character simply to avoid making the hidden badge any wider than it reasonably needs to be, considering that it will still take up space despite being visually hidden. If we use the "(placeholder)" text as you've suggested the badge could end up being wider than the title which could lead to some unexpected layouts.

Instead, I've added a comment to briefly explain this, although would be open to a shorter full word as a placeholder if you can think of a good one.

Reply:
Oh good point about the horizontal space, I hadn't thought of that. I suppose we could always apply a width of 0, with overflow hidden, but so long as it doesn't show up for screen readers and we can understand why it's there, then that works for me 👍